src/marshaller/HipieDDL.js

```diff
@@ -1060,6 +1069,15 @@
         return this;
     };
 
+    Visualization.prototype.calcRequestFor = function (visualization) {
+        var retVal = {};
+        this.getUpdatesForVisualization(visualization).forEach(function (updatesObj) {
+            //  TODO:  When we support more than "click" this will need enhancment...
```

Comment:
Yes and real one - when we support more than "click" that loop will need to decide which one to use.